`deis run` generates 500 error

[Integration tests](http://ci.deis.io/view/example-apps/job/test-integration-clojure-ring/47/console) against master found an error in `deis run`:

```
=== appssample Domains
No domains


ok
/home/jenkins/workspace/test-integration-clojure-ring/src/github.com/deis/deis/tests/example-clojure-ring
apps:run echo hello

500 INTERNAL SERVER ERROR
<h1>Server Error (500)</h1>

error at command wait
--- FAIL: TestApps (76.15 seconds)
    itutils.go:199: Failed:
        exit status 1
FAIL
exit status 1
```

--- a/controller/api/tasks.py
+++ b/controller/api/tasks.py
@@ -79,12 +79,14 @@ def stop_containers(containers):
 def run_command(c, command):
     release = c.release
     version = release.version
-    image = release.image
+    image = '{}:{}/{}'.format(settings.REGISTRY_HOST,
+                              settings.REGISTRY_PORT,
+                              release.image)
     try:
         # pull the image first
         rc, pull_output = c.run("docker pull {image}".format(**locals()))
         if rc != 0:
-            raise EnvironmentError('Could not pull image: {pull_image}'.format(**locals()))
+            raise EnvironmentError('Could not pull image: {image}'.format(**locals()))
         # run the command
         docker_args = ' '.join(['--entrypoint=/bin/sh',
                                 '-a', 'stdout', '-a', 'stderr', '--rm', image])

